recipes/graphene-django/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "graphene-django" %}
+{% set version = "2.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6e64bc9df1e4d4be247aedc63f61047cae673156198e433110bf87ae2d072a83
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - django >=1.11
+    - graphene >=2.1.7,<3
+    - graphql-core >=2.1.0,<3
+    - pip
+    - promise >=2.1
+    - python
+    - singledispatch >=3.4.0.3
+    - six >=1.10.0
+  run:
+    - django >=1.11
+    - graphene >=2.1.7,<3
+    - graphql-core >=2.1.0,<3
+    - promise >=2.1
+    - python
+    - singledispatch >=3.4.0.3
+    - six >=1.10.0
+
+test:
+  imports:
+    - graphene_django
+    - graphene_django.debug
+    - graphene_django.debug.sql
+    - graphene_django.debug.tests
+    - graphene_django.filter
+    - graphene_django.filter.tests
+    - graphene_django.forms
+    - graphene_django.forms.tests
+    - graphene_django.management
+    - graphene_django.management.commands
+    - graphene_django.rest_framework
+    - graphene_django.rest_framework.tests
+    - graphene_django.tests
+    - graphene_django.tests.issues
+    - graphene_django.utils
+  requires:
+    - coveralls
+    - django-filter <2;python_version<'3'
+    - django-filter >=2;python_version>='3'
+    - djangorestframework >=3.6.3
+    - mock
+    - pytest >=3.6.3
+    - pytest-cov
+    - pytest-django >=3.3.2
+    - pytz
+
+about:
+  home: https://github.com/graphql-python/graphene-django
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

```suggestion
  license_file: LICENSE
```

This will pull the license file from pypi where it is located.

---

Done! 🚀